ghost/magic-link/package.json

```diff
@@ -25,6 +25,8 @@
   },
   "dependencies": {
     "@tryghost/errors": "1.2.17",
+    "@tryghost/tpl": "^0.1.18",
```

Comment:
What's the convention these days for intenal packages regarding versioning? Do we allow for "loose" aka "^x.x.x" or still should be doing "strict" versioning aka "x.x.x"?

Reply:
All dependencies that appear in package.json files in this repo should be pinned to a specific version

Even if you don't commit that, Renovate will open a PR to pin them afterwards